src/main/java/edu/greenblitz/tobyDetermined/commands/NodeToNeighbourCommand.java

```diff
@@ -0,0 +1,61 @@
+package edu.greenblitz.tobyDetermined.commands;
+
+import edu.greenblitz.tobyDetermined.Nodesssss.NodeArm;
+import edu.greenblitz.tobyDetermined.Nodesssss.NodeBase;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.ElbowSub;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Extender;
+import edu.greenblitz.utils.GBCommand;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class NodeToNeighbourCommand extends GBCommand {
+    private final Extender extender;
+    private final ElbowSub elbowSub;
+    private NodeArm start;
+    private NodeArm end;
+
+
+    public NodeToNeighbourCommand(NodeArm start, NodeArm end){
+        extender = Extender.getInstance();
+        elbowSub = ElbowSub.getInstance();
+        require(elbowSub);
+        require(extender);
+        this.start = start;
+        this.end = end;
+    }
+    public static double Division(double a, double b, double gamma) {
+        double c = Math.sqrt(a*a+b*b-2*a*b*Math.cos(gamma));
```

Comment:
Raz - I would factor the cos-ine law to it's own function

Reply:
did that